packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.template.html

```diff
@@ -105,7 +105,7 @@
             data-id="volume-backup-type-{{volumeOption.id}}"
             data-name="volume-backup-type-{{volumeOption.id}}"
             data-model="$ctrl.volumeBackupModel.selected.volumeOption"
-            data-disabled="$ctrl.isCreating || !$ctrl.volumeBackupModel.selected.volume || $ctrl.volumeBackupModel.selected.volume.isLocalZone"
+            data-disabled="$ctrl.isCreating || !$ctrl.volumeBackupModel.selected.volume || $ctrl.volumePrices[volumeOption.id] === null"
```

Comment:
Can `volumePrices[volumeOption.id]` be `undefined` ?

Reply:
No, volumeOption comes from a constant, and volumePrices is mapped from the same constant, in formatVolumePrice I return null